kits/peer-demo-showcase/apps/.env.example

```
@@ -0,0 +1,18 @@
+LAMATIC_SUBMISSION_FLOW_ID=your_submission_flow_id
+LAMATIC_SUBMISSIONS_MANAGER_FLOW_ID=your_submissions_manager_flow_id
+LAMATIC_SPONSORS_MANAGER_FLOW_ID=your_sponsors_manager_flow_id
+LAMATIC_JUDGING_MANAGER_FLOW_ID=your_judging_manager_flow_id
+LAMATIC_EVENT_CONFIG_FLOW_ID=your_event_config_flow_id
+
+# Resend Email Integration (100% Free - 3,000 emails/mo)
+RESEND_API_KEY=re_your_resend_api_key
+RESEND_FROM="onboarding@resend.dev"
+
+# Security & App Config
+ADMIN_PASSWORD=coder
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)